### PR TITLE
feat: Remove feature flag templates

### PR DESCRIPTION
**Pull request by alainajaffer**

### CODE REVIEW

File: site/pages/dashboard.page.tsx [hunk 1]
```diff
@@ -256,11 +254,9 @@ const Dashboard = ({
             <Link className="govuk-link" href="/view-all-disruptions?draft=true">
                 <h2 className="govuk-heading-s text-govBlue">Draft disruptions</h2>
             </Link>
-            {stage !== "prod" && stage !== "preprod" && (
-                <Link className="govuk-link" href="/view-all-templates">
-                    <h2 className="govuk-heading-s text-govBlue">Templates</h2>
-                </Link>
-            )}
+            <Link className="govuk-link" href="/view-all-templates">
```
Comment (alvak1) [inline site/pages/dashboard.page.tsx:257]: Should we remove the feature flag after refactoring the code as per existing tasks in the to do column of Jira board?

Comment (alainajaffer) [reply site/pages/dashboard.page.tsx:257]: @lj-kpmg

Comment (alainajaffer) [reply site/pages/dashboard.page.tsx:257]: Refactoring won't impact functionality as discussed

**Review by alvak1 — APPROVED** [review]
Approved

**Pull request merged**